cost/aws/savings_plan/recommendations/aws_savings_plan_recommendations.pt

```diff
@@ -103,6 +103,8 @@ credentials "auth_aws" do
   aws_account_number $param_aws_account_number
 end
 
+auth "auth_rs", type: "rightscale"
```

Comment:
as mentioned for a previous template as part of this pull request - flexera auth should be done in the other way so that the policy can run in both NAM and EU

`credentials "auth_rs" do
  schemes "oauth2"
  label "Flexera_Automation"
  description "Select FlexeraOne OAuth2 credentials"
  tags "provider=flexera"
end`